src/pages/taskHistory/TaskHistoryRow.tsx

```diff
@@ -38,6 +42,13 @@ const TaskHistoryRow = memo((props: ListChildComponentProps) => {
               failingTests={failingTests}
               label={label}
               loading={loading}
+              onClick={(v: TaskCellAnalytics) => {
+                sendEvent({
+                  name: "Click task cell",
+                  page: "Task history",
+                  ...v,
+                });
+              }}
```

Comment:
We don't need to log page or any additional information. Since the page is logged automatically. Maybe the task status.